doozerlib/distgit.py

```diff
@@ -1135,8 +1135,15 @@ def _merge_source(self):
             if os.path.isfile("Dockerfile"):
                 os.remove("Dockerfile")
 
-            # Rename our distgit source Dockerfile appropriately
-            os.rename(dockerfile_name, "Dockerfile")
+            try:
+                # Rename our distgit source Dockerfile appropriately
+                os.rename(dockerfile_name, "Dockerfile")
+            except OSError as err:
+                print("TODO: Print out what dang image this is and a link to the source on github, if that's not too hard.")
+                print("Expected dockerfile '{}' does not exist, but these do!")
```

Comment:
This would help at the very least
```suggestion
                print("Expected dockerfile '{}' does not exist, but these do!".format(os.path.realpath(os.curdir)))
```

Reply:
`self.distgit_dir` is what you should use. Nothing should ever be dependent on `pwd` here.